File: Java/Conversoes/QuestionFour.java
package br.com.fiap.Conversoes;
import java.util.Scanner;

public class QuestionFour {
    public static void main(String[] args){

        Scanner collector = new Scanner(System.in);

        System.out.println("Digite um número qualquer:");
        int number = collector.nextInt();

        int calculo = number % 2;

        if (number > 0 && calculo == 0) {
            System.out.println("Seu número " + number + " é positivo e par!");
        } else if (number > 0 && calculo > 0) {
            System.out.println("Seu número " + number + " é positivo e ímpar.");
        } else if (number < 0 && calculo == 0) {
            System.out.println("Seu número " + number + " é negativo e par.");
        } else {
            System.out.println("Seu número " + number + " é negativo e ímpar.");
        }
    }
}

File: Python/QuestionFour.py
numero = int(input("Digite um número desejado: "))
divisao_par = numero % 2

if numero > 0 and divisao_par == 0:
    print(f"\nO número {numero} é positivo e par")
elif numero > 0 and divisao_par != 0:
    print(f"\nO número {numero} é positivo e ímpar")
elif numero < 0 and divisao_par == 0:
    print(f"\nO número {numero} é negativo e par")
elif numero < 0 and divisao_par != 0:
    print(f"\nO número {numero} é negativo e ímpar")
elif numero == 0:
    print(f"O número {numero} é zero e par")